src/extension/tab/helpers.ts

```diff
@@ -46,6 +46,7 @@ export interface SerializedError {
 }
 
 export type QueryInfo = {
```

Comment:
Could we rename this type at some point? It's a bit of a red herring since we have a type of the same name over in AC.